packages/hydrogen/src/with-cache.example.ts

```diff
@@ -13,7 +13,8 @@ export default {
     const cache = await caches.open('my-cms');
     const withCache = createWithCache({
       cache,
-      waitUntil: executionContext.waitUntil,
+      waitUntil: executionContext.waitUntil.bind(executionContext),
```

Comment:
Are we losing `this` context without the `bind`?

Reply:
Yes, we already use `.bind` in all our templates so I just updated it here as well.